src/modv/draw.js

```diff
@@ -90,14 +93,26 @@ function draw(δ) {
         }
       }
 
-      if (!enabled || alpha === 0) return
+      if (!enabled || alpha === 0) {
+        continue
+      }
 
-      Layer.moduleOrder.forEach((moduleName, moduleIndex) => {
+      const moduleOrderLength = Layer.moduleOrder.length
+      for (
+        let moduleIndex = 0;
+        moduleIndex < moduleOrderLength;
+        ++moduleIndex
+      ) {
+        const moduleName = Layer.moduleOrder[moduleIndex]
         const Module = store.getters['modVModules/outerActive'][moduleName]
 
-        if (!Module) return
+        if (!Module) {
```

Comment:
Why did this change from `please don't go on` to `let's continue`?

Reply:
In `forEach` loop a `return` acts similarly to a `continue` in a `for` loop.

If I had left the `return` in there, any Module with no opacity, that wasn't enabled or any layer with no modules would have stopped the rest of the draw loop entirely.